src/fetch_purchases_suppliers.py

```diff
@@ -0,0 +1,40 @@
+import requests
+import pandas as pd
+
+
+BASE_DATA_DIR = './data/'
+
+def fetch_suppliers():
+    raw_suppliers = []
+    first_response = requests.get('http://compras.dados.gov.br/fornecedores/v1/fornecedores.json').json()
+    raw_suppliers = raw_suppliers + first_response['_embedded']['fornecedores']
+
+    last_page = first_response['count']
+    for page in range(2, last_page):
+        print('{}/{}'.format(page, last_page))
+        response = requests.get('http://compras.dados.gov.br/fornecedores/v1/fornecedores.json').json()
+        raw_suppliers = raw_suppliers + first_response['_embedded']['fornecedores']
+    return raw_suppliers
+
+
+def prepare_data(raw_data):
+    suppliers_attributes_columns = {'id': 'id', 'cnpj': 'cnpj', 'nome': 'name', 'ativo': 'active',
+    'recadastrado': 'relisted', 'id_municipio': 'city_id', 'uf': 'state', 'id_natureza_juridica': 'legal_nature_id',
+    'id_porte_empresa': 'company_size_id', 'id_ramo_negocio': 'business_id', 'id_unidade_cadastradora': 'responsible_entity',
+    'id_cnae': 'cnae_id', 'habilitado_licitar': 'allowed_to_bid'}
+
+    df = pd.DataFrame(raw_data, columns = suppliers_attributes_columns.keys())
+
+    return df
+
+
+def save_csv(df):
+    df.to_csv(path_or_buf=BASE_DATA_DIR+'purchase_suppliers.xz', sep=',', compression='xz', encoding='utf-8', index=False)
```

Comment:
We have the practice of adding a date prefix to every dataset (`YYYY-MM-DD`) as a way of versioning our datasets (and consequentially our scripts and notebooks). Would you mind adding that?